pkg/trace/stats/concentrator.go

```diff
@@ -201,7 +206,8 @@ func (c *Concentrator) addNow(pt *traceutil.ProcessedTrace, containerID string)
 			b = NewRawBucket(uint64(btime), uint64(c.bsize))
 			c.buckets[btime] = b
 		}
-		b.HandleSpan(s, weight, isTop, pt.TraceChunk.Origin, aggKey, c.peerSvcAggregation)
+
+		b.HandleSpan(s, weight, isTop, pt.TraceChunk.Origin, aggKey, c.peerSvcAggregation, c.customTags, c.customTagSpanNames)
```

Comment:
Note for code above: if the span name is in the custom tag config by span name, we should calculate stats.
Shouldn't matter if it's top-level/measured or has the right span.kind.